common/oatbox/extension/Manifest.php

```diff
@@ -81,12 +82,16 @@ public function __construct($filePath, ComposerInfo $composerInfo = null)
     {
         // the file exists, we can refer to the $filePath.
         if (!is_readable($filePath)) {
-            throw new ManifestNotFoundException("The Extension Manifest file located at '${filePath}' could not be read.");
+            throw new ManifestNotFoundException(
+                "The Extension Manifest file located at '${filePath}' could not be read."
+            );
         }
         $this->manifest = require($filePath);
         // mandatory
         if (empty($this->manifest['name'])) {
-            throw new exception\MalformedManifestException("The 'name' component is mandatory in manifest located at '{$this->filePath}'.");
+            throw new exception\MalformedManifestException(
```

Comment:
suggestion: I'd be better to add a `use` statement for this one instead to be consistent with the usages of other exception types from the `oat\oatbox\extension` namespace

Reply:
I see this was already there (the only change is in formatting), feel free to keep it as it is :)

Reply:
I would swear it is not :D, otherwise phpstorm would have suggested me to remove it. I see other exceptions but not this particular one. Im going to leave it as it is for now since i dont want to disturb the backport but for the next one i will be for sure aware. This second review also allowed me to notice files contains unused includes too. Thanks hector! :)